Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -438,26 +440,38 @@
 
     replaceContentsOfRangeInSession(uuid, resolvedRange, *fragment);
 
-    auto sessionRangeAfterReplace = contextRangeForSessionWithUUID(uuid);
-    if (!sessionRange) {
-        ASSERT_NOT_REACHED();
-        return;
-    }
     auto characterRangeAfterReplace = WebCore::CharacterRange(range.location, range.length + characterCountDelta);
 
     bool attachedToExistingSession = false;
     for (auto& session : m_textIndicatorCharacterRangesForSessions) {
         if (session.first == uuid) {
             // session.second.last() will always be valid because when creating
             // the structure we always create the vector with an element.
-            session.second.append({ replaceTextIndicatorUUID, subCharacterRange(range, session.second.last().second) });
+            session.second.append({ sourceTextIndicatorUUID, subCharacterRange(range, session.second.last().second) });
```

Comment:
can
```
subCharacterRange(range, session.second.last().second)
```
be cached in a separate variable since it's used twice in these two lines?